src/Tool/BinaryDecoder/CMakeLists.txt

```diff
@@ -2,8 +2,8 @@
 
 add_executable(binary-decoder BinaryDecoder.cpp)
 target_link_libraries(binary-decoder
```

Comment:
target_link_libraries(binary-decoder
  PRIVATE
  LLVMSupport
  onnx
  )

is enough